pkg/cmd/roachtest/tests/kvbench.go

```diff
@@ -184,13 +184,6 @@ func registerKVBench(r registry.Registry) {
 	}
 }
 
-func makeKVLoadGroup(c cluster.Cluster, numRoachNodes, numLoadNodes int) loadGroup {
```

Comment:
Good riddance! Do we know why this parameterized by `numLoadNodes` even though the current implementation hardcodes `1`?

Reply:
Not sure, I believe there were other cases of tests doing this. Perhaps future proofing?